acceptance-tests/pom.xml

```diff
@@ -356,14 +356,14 @@
                                     <platform>linux</platform>
                                     <bit>64</bit>
                                     <version>${chromedriver.version}</version>
-                                    <url>https://chromedriver.storage.googleapis.com/${chromedriver.version}/chromedriver_linux64.zip</url>
+                                    <url>https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/${chromedriver.version}/linux64/chromedriver-linux64.zip</url>
```

Comment:
The chromedriver.storage site does not have the required version of chromedriver. https://chromedriver.chromium.org/downloads suggests to use the other source for 115.0.5790.102